### PR TITLE
Improves the SpinnerComponent

### PR DESCRIPTION
**Pull request by GabrielInTheWorld**
- The AppComponent listens to `stable`-state and `user` to determine whether the spinner should be hidden.

### CODE REVIEW

File: client/src/app/app.component.ts [hunk 1]
```diff
@@ -88,10 +99,24 @@ export class AppComponent {
...
-                spinnerService.setVisibility(false);
+            .subscribe(stable => {
+                // check the stable state only once.
+                if (!this.isStable && stable) {
+                    this.isStable = true;
+                    this.checkConnectionProgress();
+                }
             });
+        // subscribe, to get the user.
+        operator.getViewUserObservable().subscribe(user => {
```
Comment (FinnStutzenstein) [inline client/src/app/app.component.ts]: Please be careful here! I guess, that if the anonymous is enabled, `user` will always stay `null`... Please try this out. If so, I can give you hints, where to get these information (users loaded finished) from somewhere else.

Comment (GabrielInTheWorld) [reply client/src/app/app.component.ts]: You're right, the user will always be `undefined`... Can you give me any hints for this, please? Thanks!

Comment (GabrielInTheWorld) [reply client/src/app/app.component.ts]: **Update**: I have pushed a second commit - how do you think about my solution for this?

**Comment by GabrielInTheWorld** [comment]
@emanuelschuetze Done

**Pull request merged**